fecgov/fec-cms
# Add missing migration for CMS

## Summary (required)

- Resolves #6342 

- Create and commit the missing migration so working GitHub branches are up to date with production
- See the issue screenshots for verification that this migration already exists on  prod

**Future work:**
- [ ] Configure circle to show the migrations during build. cc @pkfec  

### Required reviewers

one frontend OR backend dev

## Impacted areas of the application

migrations to cms database


## Related PRs

https://github.com/fecgov/fec-cms/pull/6225

## How to test

- checkout branch and run `./manage.py migrate`
- run .`/manage.py makemigrations --dry-run --verbosity 3`
    - It should return: "no changes detected"
